lib/wallet/http.js

```diff
@@ -1693,5 +1693,5 @@ function enforce(value, msg) {
 /*
  * Expose
  */
-
+HTTP.TransactionOptions = TransactionOptions;
```

Comment:
dude, sorry to be super annoying about this, but the newline should stay in :-)
I think when you squashed you might've removed the commit that added the newline.